api-reference/endpoint/query.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "Check Software EOL"
+api: "POST https://eol.xeol.com/api/v1/query"
+description: "This endpoint checks if the provided software identifier(s) are end-of-life"
+---
+
+### Body
+
+<ParamField body="identifiers" type="object[]">
```

Comment:
instead of identifiers, we'll need field for the specific identifiers (CPE, PURL, or custom).

The way this is now we won't be able to handle custom identifiers.